visallo/visallo
# add placeholder text for end date in notification and i18n other placeholder text

Testing Instructions:
- Make sure create notification admin pane has a placeholder text in every field except start date

Points of Regression: Admin Create Notification Pane

CHANGELOG
Added: Placeholder text for End Date Field in admin create notification pane
Changed: Placeholder text to i18n in admin create notification pane
